### PR TITLE
[#794] Export Metadata Subfields Publisher And Publication Date

### PR DESCRIPTION
**Pull request by Meowcenary**
Another simple and to the point PR. This has been rebase off of 795 Subfield Subjects Redux so review that first.

### CODE REVIEW

File: app/models/invenio_rdm_record_converter.rb [hunk 1]
```diff
@@ -90,6 +90,8 @@ def invenio_metadata(gf)
       "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} },
       "description": gf.description.first,
       "additional_descriptions": gf.description.last(gf.description.size-1).map{ |add_desc| {"description": add_desc, "type": "other", "lang": "eng"} },
+      "publisher": gf.publisher.shift,
```
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:93]: If no publisher present, `gf.publisher.shift` returns `nil` which responds to `to_json` with `"null"`. We can change this to something else if we want. Maybe an empty string?

**Comment by Meowcenary** [comment]
Alrighty, I've cleared out some PRs so now I'm going to update this. It will be ready for review after that's done in a moment.

File: app/models/invenio_rdm_record_converter.rb [hunk 2]
```diff
@@ -90,6 +90,8 @@ def invenio_metadata(gf)
...
+      "publication_date": "#{gf.date_uploaded.month}-#{gf.date_uploaded.day}-#{gf.date_uploaded.year}",
```
Comment (sharpattack) [inline app/models/invenio_rdm_record_converter.rb:94]: For invenioRDM, should this be YEAR-MONTH-DAY?

**Review by sharpattack — APPROVED** [review]
Another simple PR, thanks. I will approve now for speed, but double-check the date format. I see it differently on the invenioRDM docs.

File: app/models/invenio_rdm_record_converter.rb (hunk 2 above)
Comment (Meowcenary) [reply app/models/invenio_rdm_record_converter.rb:94]: You're right. I think I made this change before the follow up with just the `date_created` field. I'll fix this up before merging.

**Comment by Meowcenary** [comment]
Running tests for this and if that looks good I'll merge, run the export on stage, and put the results on slack or in an email for everyone.

**Pull request merged**